### DISCUSSION TITLE
Welcome to ARSoftware.ManejadorDocumentosCfdi Discussions!

### DISCUSSION THREAD

**Discussion by AndresRamos** [AndresRamos/ARSoftware.ManejadorDocumentosCfdi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rossda** [comment]
Hola de casualidad vas a actualizar el manejador de documentos, con el nuevo release del webservice del SAT?

**Reply by AndresRamos** [reply]
Estoy trabajando primero en la [ARSoftware.Cfdi.DescargaMasiva](https://github.com/AndresRamos/ARSoftware.Cfdi.DescargaMasiva) para luego actualizar esta aplicación.

**Reply by rossda** [reply]
Muchas gracias.... ya lo hice funcionar... pero todas las peticiones siguen en estatus 1, espero que sea por fin de mes

**Reply by AndresRamos** [reply]
A que te refieres que lo hiciste funcionar?
